src/main/webapp/loginRep.html

```diff
@@ -3,17 +3,21 @@
   <head>
     <meta charset="UTF-8">
     <title>Representative Login Page</title>
-    <link rel="stylesheet" href="style.css">
```

Comment:
We don't seem to have a style.css? I might be missing something but this is giving me error messages when I run in dev